copyparty/web/browser.js

```diff
@@ -338,6 +338,14 @@ var Ls = {
 		"f_empty": 'this folder is empty',
 		"f_chide": 'this will hide the column «{0}»\n\nyou can unhide columns in the settings tab',
 		"f_bigtxt": "this file is {0} MiB large -- really view as text?",
+
+		formatSize: function(bytes) {
+			if (bytes < 1024) return bytes + ' B';
+			let k = 1024;
+			let sizes = ['B', 'KB', 'MB', 'GB', 'TB'];
```

Comment:
As you are using 1024 multiples, I recommbed the "-ibi" suffixes instead.
So Kibibyte (KiB), Mebibyte (MiB), and so on.

[More info](https://en.wikipedia.org/wiki/Byte?useskin=vector#Multiple-byte_units)